packages/worker/src/asg.ts

```diff
@@ -1,22 +1,17 @@
-import {
-	AutoScalingClient,
-	SetInstanceProtectionCommand,
-} from '@aws-sdk/client-auto-scaling';
+import { SetInstanceProtectionCommand } from '@aws-sdk/client-auto-scaling';
 import { readFile } from '@guardian/transcription-service-backend-common';
+import { getASGClient } from '@guardian/transcription-service-backend-common/src/asg';
```

Comment:
encapsulation of backend-common is a bit better if we import from `transcription-service-backend-common`. Then we can reorganize modules in transcription-service-backend-common without changing this import.
```suggestion
import { readFile, getASGClient } from '@guardian/transcription-service-backend-common';
```